src/test/resources/features/articles/editdeleteearticles.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Edit - Delete Articles
```

Comment:
I would split these feature into two: `edit.feature` and `delete.feature`.

Reply:
Right, I'll split the feature